test/development/acceptance-app/__snapshots__/ReactRefreshLogBox.test.ts.snap

```diff
@@ -101,10 +101,10 @@ exports[`ReactRefreshLogBox app default should strip whitespace correctly with n
 `;
 
 exports[`ReactRefreshLogBox app turbo Can't resolve @import in CSS file 1`] = `
-"./app/styles2.css:1:9
+"./app/styles2.css:1:2
 Module not found: Can't resolve './boom.css'
 > 1 | @import "./boom.css"
-    |         ^^^^^^^^^^^^
+    |  ^
```

Comment:
The spans where better before

Reply:
`lightningcss` does not provide any span information to fix this regression.
It only stores the starting location.